README.md

```diff
@@ -64,3 +68,33 @@ For more information, see the [Rust API embedding chapter] of the Wasmtime docum
 [Rust API embedding chapter]: https://bytecodealliance.github.io/wasmtime/embed-rust.html
 
 It's Wasmtime.
+
+#### Standalone runtime
+
+The [Releases](https://github.com/bytecodealliance/wasmtime/releases) page has pre-built binaries for each release, with binaries for Linux, macOS and Windows.
+
+If you want to build the latest Wasmtime from scratch, you will need:
+
+* An updated [Rust toolchain](https://www.rust-lang.org/install.html) installed
+* A C toolchain installed - e.g. the [`build-essential`](https://packages.ubuntu.com/xenial/build-essential) package in Ubuntu
+* A clone of the Wasmtime Git repo
+
+Example:
+
+```sh
+git clone https://github.com/bytecodealliance/wasmtime.git
+git submodule update --init --recursive
```

Comment:
I think this can be removed by adding `--recursive` to the `git clone` command above